src/helpers/metrics.MD

```diff
@@ -0,0 +1,22 @@
+# Metrics
```

Comment:
Do we need to use uppercase `MD` for the extension?

Reply:
Oh, we do not - I don't know why I did that lol